test/src/xquery/xquery3/suite.xql

```diff
@@ -9,5 +9,6 @@ test:suite((
     inspect:module-functions(xs:anyURI("concat.xql")),
     inspect:module-functions(xs:anyURI("groupby.xql")),
     inspect:module-functions(xs:anyURI("flwor.xql")),
-    inspect:module-functions(xs:anyURI("typeswitch.xql"))
+    inspect:module-functions(xs:anyURI("typeswitch.xql")),
+    inspect:module-functions(xs:anyURI("fn.xql"))
```

Comment:
My idea is to have a script here For all kind of tests on xquery fn functions 

I already spend quite some time in finding a better name. So the intention is not to have folding tests only.

That said.... Which test target does start the xquery tests?